UKHomeOffice/nmsw-ui
# NMSW-436 [pt5] Hande FAL1 General declaration missing mandatory fields inc Sorting

# Ticket

NMSW-436


----

## AC

Given I have uploaded a FAL 1 file
When there are required fields with null values
Then I am shown an error message to help me fix them

----

## To test

- create a voyage
- upload FAL1 sheet from Jira ticket
- click submit
- > you should see the errors listed in the Jira ticket
- > the errors should be in the order of 'row' then 'column' (see image on Jira ticket for confirmation)

----

## Notes
